Libraries/NewAppScreen/index.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import React, {Fragment} from 'react';
+import {
+  StyleSheet,
+  ScrollView,
+  View,
+  Text,
+  Platform,
+  StatusBar,
+  SafeAreaView,
+} from 'react-native';
+
+import Header from './components/Header';
+import LearnMoreLinks from './components/LearnMoreLinks';
+import Colors from './components/Colors';
+
+const Section = ({children}) => (
+  <View style={styles.sectionContainer}>{children}</View>
+);
+
+const ReloadInstructions = () => {
+  return Platform.OS === 'ios' ? (
+    <Text style={styles.sectionDescription}>
+      Press <Text style={styles.highlight}>Cmd+R</Text> in the simulator to
+      reload your app's code
+    </Text>
+  ) : (
+    <Text style={styles.sectionDescription}>
+      Double tap <Text style={styles.highlight}>R</Text> on your keyboard to
+      reload your app's code
+    </Text>
+  );
+};
+
+const DebugInstructions = () => {
+  return Platform.OS === 'ios' ? (
+    <Text style={styles.sectionDescription}>
+      Press <Text style={styles.highlight}>Cmd+D</Text> in the simulator or{' '}
+      <Text style={styles.highlight}>Shake</Text> your device to open the React
+      Native debug menu.
+    </Text>
+  ) : (
+    <Text>
+      Press <Text style={styles.highlight}>menu button</Text> or
+      <Text style={styles.highlight}>Shake</Text> your device to open the React
+      Native debug menu.
+    </Text>
+  );
+};
+
+const App = () => {
+  return (
+    <Fragment>
+      <SafeAreaView style={styles.topSafeArea} />
```

Comment:
I'm still a bit confused why we need two `SafeAreaView`s. My understanding is that the `SafeAreaView` with the children should handle both top and bottom safe area insets.

Reply:
@eliperkins I used two safe areas because I wanted the top safe area with the color of the header and the bottom with the color of the screen background. There is another way to do this?

Reply:
Ah, interesting! I hadn't thought about that use-case. This totally makes more sense why we would reach for multiple `<SafeAreaView>`s now.

One option could be to use `ScrollView`'s `contentInsetAdjustmentBehavior`: https://facebook.github.io/react-native/docs/scrollview.html#contentinsetadjustmentbehavior

To do this, we would have to move the `ScrollView` up to be the root view (that is, outside of the `SafeAreaView` this is currently containing it) and this will let UIKit take over on adjusting the insets for the safe areas. Then, by setting the background of the `ScrollView` to be `Colors.lighter`, we could accomplish this all without `SafeAreaView` at all!

I put a Snack up of this here: https://snack.expo.io/@eliperkins/lonely-churros

The downside to this approach is that the end of our scroll view would then be the grey color as well. I think the trade off is better than over-prescribing `<SafeAreaView>`s!

Reply:
This could also use `Animated` to change the color of the `<ScrollView>`'s `backgroundColor` while scrolling, but that seems a bit too complex for a sample app.